### DISCUSSION TITLE
Blade @class directive renders class attribute even if its empty

### DISCUSSION THREAD

**Discussion by stoffpalette** [laravel/framework#43947] [General]
Blade's @class directive should not render a class attribute, if every class-name is evaluated to false.

Currently, if you do `@class(['first-class' => false, 'second-class' => false])` the rendered result is `"class"` instead of `""`.

**Comment by angeljqv** [comment]
It doesn't break anything

**Comment by stoffpalette** [comment]
No, it doesn't break if used standalone, but it is not expected and needless. Furthermore it is easy to fix.

**Comment by angeljqv** [comment]
> it is easy to fix.

Feel free to make a PR
